st2mistral/functions/stackstorm.py

```diff
@@ -18,17 +18,25 @@
 from oslo_config import cfg
 from oslo_log import log as logging
 
-from st2mistral import config
-config.register_opts()
-
 from mistral import exceptions as exc
 from st2mistral.utils import http
 
+from st2mistral import config
+config.register_opts()
```

Comment:
This was to address the "module level import not at top of file" PEP8 violation, though I can see now that flake8 doesn't care for some reason (even though E402 is not being ignored).

Anyways, I can revert this, just part of the overall style fixes.